spec/requests/api/v1/checkins_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+require 'byebug'
+
+RSpec.describe CheckinsController, :type => :controller do
+  render_views
+
+  let(:json) { JSON.parse(response.body) }
+
+
+  describe "Checkins API" do
+    before do
+      @user1 = User.create(name:"User 01", email:"test@test.es", uid: 1518243131643898)
```

Comment:
you could extract to a let like `let!(:user1) {User.create(name:"User 01", email:"test@test.es", uid: 1518243131643898)}`